### DISCUSSION TITLE
Can I read and/or write Apple II disks?

### DISCUSSION THREAD

**Discussion by junkjunker** [keirf/greaseweazle#205] [Q&A]
I've got a ton of Apple II disks that I'd like to archive and be able to use with emulators.
I also have a TEAC FD-55GFR 5.25" floppy drive.
Will the GreaseWeazle board let me at least read the Apple II disks, and hopefully write to them as well?
THANKS

**Comment by keirf** [comment]
Should do. The fluxengine project can even decode apple ii I believe, and it supports greaseweazle.

**Reply by Blazingstix** [reply]
This is also something I'm interested in.  I have a TEAC FD-155GF, combo 3.5/5.25 floppy drive, and I just bought an old Apple II game.  I have no experience with the Apple II (only DOS from that era).

From what I've read, these disks are most likely "flippy" disks encoded with GCR (instead of MFM) and each side has 35 tracks each with 16 sectors of 256 bytes, for a total of 143,360 bytes of data.

I'm unsure what commands to pass gw.exe so it can capture them properly to an scp image file.   No matter what commands I try, it seems to get the 1st side well enough (plenty of green in the HxCFloppyEmulator track analysis (a few slivers of red), but I get almost solid red for the other side.

Can somebody help with the proper arguments to try?

(Edit to add: I'm using a GreaseWeazle v4, with FW 1.1)

**Reply by junkjunker** [reply]
The conversation on the Facebook page indicates that reading Apple II disks with the Greaseweazle software is problematic, but the Fluxengine software can use the Greaseweazle hardware and get good reads:
https://www.facebook.com/groups/greaseweazle/posts/5379298738822370/

**Reply by Blazingstix** [reply]
Thank you. I didn't realize Fluxengine had fully separate software and hardware components.

**Comment by markaudacity** [comment]
What about writing them?
